### ISSUE TITLE
admin console change 'Master Broker' does not show/track the default master broker instance in JMS Service config screen in Configurations -> a cluster config

### ISSUE THREAD

**Issue by amykang2020** [javaee/glassfish#22050]
In GlassFish admin console
1. create a cluster cluster1
2. create inst1, then inst2 in the cluster
3. start the cluster
4. go to Configurations -> cluster1 config -> JMS Service

In the JMS Service screen, the field 'Master Broker' is empty.  It should be 'inst1' as that is the default GlassFish master broker instance.

**Comment by sumauppa** [comment]
@amykang2020 How do we get the default value of a master broker for cluster config?
After 4 steps mentioned in the issue, agree that console is not showing any value.
We cannot get a value through CLI too using 
./glassfish5/glassfish/bin/asadmin get configs.config.cl2-config.jms-service.master-broker. 
remote failure: Dotted name path configs.config.cl2-config.jms-service.master-broker not found.
Command get failed.

I tried searching through cluster also to find if there is any command to get the default value. I did not find one. 
The above command and console works, once we execute change-master-broker command.

From the findings, I feel that console is working as expected.
